src/blueapi/worker/task_worker.py

```diff
@@ -86,7 +85,7 @@ class TaskWorker:
     _state: WorkerState
     _errors: list[str]
     _warnings: list[str]
-    _task_channel: Queue  # type: ignore
+    _task_channel: Queue
```

Comment:
Could: This type annotation can probably be parametrized now. It couldn't before, while we were supporting Python3.9

Reply:
As of 3.11, Queue is not Generic.

Reply:
Boo